app/tests/spec/views/sign_up.js

```diff
@@ -830,6 +830,32 @@ function (chai, $, sinon, p, View, CoppaDatePicker, Session, AuthErrors, Experim
                                 'signup.customizeSync.true'));
             });
         });
+
+        it('checkbox is visible for service=sync', function () {
+          return setupCustomizeSyncTest('sync', true)
+            .then(function () {
+              assert.isTrue(view.$el.find('#customize-sync').length > 0);
+            });
+        });
+
+        it('checkbox is not visible if no showChooseWhatToSyncCheckbox capability', function () {
+          broker.setCapability('showChooseWhatToSyncCheckbox', false);
+          view.context();
+          return setupCustomizeSyncTest('sync', true)
+            .then(function () {
+              assert.isFalse(view.$el.find('#customize-sync').length > 0);
+            });
+        });
+
+        it('checkbox is not visible if Choose What to Sync Web is available', function () {
```

Comment:
With `chooseWhatToSync` logic [encoded into the broker capabilities](https://github.com/mozilla/fxa-content-server/pull/3142/files#r41975309), this test will not be necessary.